cmd/poller/collector/collector.go

```diff
@@ -339,6 +343,7 @@ func (me *AbstractCollector) Start(wg *sync.WaitGroup) {
 			} else if me.Schedule.IsStandBy() {
 				// recover from standby mode
 				me.Schedule.Recover()
+				retryDelay = 1
```

Comment:
good catch!
I had completely overseen this..

Reply:
all credit to hashi825 for noticing and pointing out - i did the easy part 😄